### PR TITLE
Add a better readme to Publishing Checklist

### PR DESCRIPTION
**Pull request by davisshaver**
For https://github.com/fusioneng/publishing-checklist/milestones/v0.1.0, we should have a decent readme on this plugin.

See https://github.com/fusioneng/publishing-checklist/issues/6 and https://github.com/fusioneng/publishing-checklist/issues/8

### CODE REVIEW

**Comment by danielbachhuber** [comment]
`readme.txt` is the one you want to edit, and then you can create `readme.md` with `grunt readme`

**Comment by davisshaver** [comment]
@danielbachhuber could you review this as well? I'll come back to add screenshots and changelog as the last steps before relase.

File: readme.txt [hunk 1]
```diff
@@ -1,114 +1,76 @@
-=== Publishing-checklist ===
```
Comment (danielbachhuber) [inline readme.txt]: This file needs to be in WordPress.org markdown. Not sure why you changed it...

Comment (danielbachhuber) [inline readme.txt]: https://wordpress.org/plugins/about/readme.txt

Comment (danielbachhuber) [inline readme.txt]: https://wordpress.org/plugins/about/validator/

File: readme.txt [hunk 2]
```diff
@@ -1,114 +1,76 @@
...
-Markdown uses email style notation for blockquotes and I've been told:
-> Asterisks for *emphasis*. Double it up  for **strong**.
-
-`<?php code(); // goes in backticks ?>`
+# Publishing Checklist #
+**Contributors:** fusionengineering, danielbachhuber, davisshaver  
+**Tags:** editorial, checklist, publishing, preflight
+**Requires at least:** 4.2  
+**Tested up to:** 4.2  
+**Stable tag:** 0.1.0
```
Comment (danielbachhuber) [inline readme.txt]: Please don't change the stable tag until the stable version is released.

File: readme.txt [hunk 3]
```diff
@@ -1,114 +1,76 @@
...
+## Description ##
+
+Each time a user saves a post, Publishing Checklist validates that post type's list of tasks to make sure the content is ready for release. Tasks are validated with callbacks you supply when registering tasks.
+
+## Installation ##
+It's a plugin! Install it like any other. 
+
+Once you've done so, you'll need to register the checklist items and callbacks for your site. Here's a simple example that checks for a featured image.
+
+### integrations/class-publishing-checklist.php
```
Comment (danielbachhuber) [inline readme.txt]: Can we link to a demo on the wiki so it's easier to keep up to date?

Comment (davisshaver) [inline readme.txt]: @danielbachhuber A demo plugin would be a nice followup. I'm not sure that making a new page keeps the documentation any easier to update. I like the PR workflow of using the readme. Thoughts?

Comment (danielbachhuber) [inline readme.txt]: > I'm not sure that making a new page keeps the documentation any easier to update.

It does https://github.com/fusioneng/Shortcake/issues/250

**Comment by davisshaver** [comment]
@danielbachhuber updated to conform to .org standards. Still needs Screenshots, changelog, and FAQ. FAQ can be bare-bones. I'm going to do Screenshots and changelog last after we merge https://github.com/fusioneng/publishing-checklist/pull/15

**Pull request merged**